### ISSUE TITLE
Integrate Lottie Web Player

### ISSUE THREAD

**Issue by trieloff** [adobe/helix-embed#147]
From URL https://lottiefiles.com/16313-favorite look for a line containing

```html
<lottie-listings height="150px" type="public" src="https://assets6.lottiefiles.com/packages/lf20_5z9SbI.json" slug="16313-favorite" speed="1" bg="#ffffff" is-listing='true'></lottie-listings>
```

which has the asset URL for the web player at 

see https://lottiefiles.com/web-player?lottie_url=https://assets6.lottiefiles.com/packages/lf20_5z9SbI.json the resulting HTML looks like this:

```html
<script src="https://unpkg.com/@lottiefiles/lottie-player@latest/dist/lottie-player.js"></script>
<lottie-player
    src="https://assets6.lottiefiles.com/packages/lf20_5z9SbI.json"  background="transparent"  speed="1"   loop controls autoplay >
</lottie-player>
```

/cc @kptdobe

**Comment by kptdobe** [comment]
What would be the corresponding Markdown to use LottieFiles ?

**Comment by trieloff** [comment]
```markdown
Look at my cool animation:

 https://lottiefiles.com/16313-favorite

Don't you like it?
````

**Comment by MarquiseRosier** [comment]
Best case: web player preview
Fall Back Case: iframely image preview
Worst Case: Markdown Link renders to Link

**Comment by trieloff** [comment]
Fixed in #165
